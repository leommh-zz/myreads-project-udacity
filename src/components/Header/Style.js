const style = {
  main: {
    flex: 1,
    backgroundColor: "#000"
  },
  title: {
    fontSize: 50,
    color: "#fff"
  }
};

export default style;
