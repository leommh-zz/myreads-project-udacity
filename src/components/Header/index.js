import React from 'react';
import { Grid, Typography } from '@material-ui/core';
import Style from './Style';

const Header = () => (
    <Grid container direction="row">
        <Grid item>
            <Typography variant="subtitle1" style={Style.title}> 
                MyReads
            </Typography> 
        </Grid>
    </Grid>
)

export default Header;