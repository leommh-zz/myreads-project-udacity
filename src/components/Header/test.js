import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe('[Component] Header', () => {
  
    it('Display Header Component', () => {
        const wrapper = shallow(<Header />);
        expect(wrapper).toMatchSnapshot();
    });
    
});