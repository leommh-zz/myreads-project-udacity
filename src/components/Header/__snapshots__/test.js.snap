// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] Header Display Header Component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <WithStyles(Grid)
        item={true}
>
        <WithStyles(Typography)
                style={
                        Object {
                                "color": "#fff",
                                "fontSize": 50,
                              }
                }
                variant="subtitle1"
        >
                MyReads
        </WithStyles(Typography)>
</WithStyles(Grid)>,
      "container": true,
      "direction": "row",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Typography)
          style={
                    Object {
                              "color": "#fff",
                              "fontSize": 50,
                            }
          }
          variant="subtitle1"
>
          MyReads
</WithStyles(Typography)>,
        "item": true,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "MyReads",
          "style": Object {
            "color": "#fff",
            "fontSize": 50,
          },
          "variant": "subtitle1",
        },
        "ref": null,
        "rendered": "MyReads",
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <WithStyles(Grid)
          item={true}
>
          <WithStyles(Typography)
                    style={
                              Object {
                                        "color": "#fff",
                                        "fontSize": 50,
                                      }
                    }
                    variant="subtitle1"
          >
                    MyReads
          </WithStyles(Typography)>
</WithStyles(Grid)>,
        "container": true,
        "direction": "row",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <WithStyles(Typography)
            style={
                        Object {
                                    "color": "#fff",
                                    "fontSize": 50,
                                  }
            }
            variant="subtitle1"
>
            MyReads
</WithStyles(Typography)>,
          "item": true,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "MyReads",
            "style": Object {
              "color": "#fff",
              "fontSize": 50,
            },
            "variant": "subtitle1",
          },
          "ref": null,
          "rendered": "MyReads",
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
